data/channels.csv

```diff
@@ -3809,6 +3811,7 @@ CDirect.fr,CDirect,,,,FR,,,c/FR,fra,,FALSE,,,,,https://i.imgur.com/Zis4TRh.png
 CDMInternacional.pr,CDM Internacional,,,,PR,,,c/PR,spa;eng,,FALSE,,,,,https://i.imgur.com/jOpammm.png
 CDN.do,CDN,,,,DO,,,c/DO,spa,,FALSE,,,,https://cdn.com.do/,https://i.imgur.com/eQyjIgf.png
 CDNDeportes.do,CDN Deportes,,,,DO,,,c/DO,spa,,FALSE,,,,https://cdndeportes.com.do/,https://i.imgur.com/yU5LqTL.png
+CDNews.in,CD News,,,,IN,,,c/IN,mal,news,FALSE,,,,,https://i.imgur.com/VIPPsxf.png

```

Comment:
```suggestion
CDNews.in,CD News,,,,IN,,,c/IN,mal,news,FALSE,,,,,https://i.imgur.com/zIXsuC2.png
```
Removed the transparent background around the logo.

Reply:
It already had transparency!